src/doppler/sinks/syslogwriter/syslog_writer_test.go

```diff
@@ -145,28 +117,43 @@ var _ = Describe("SyslogWriter", func() {
 			sysLogWriter.Close()
 		})
 
-		It("gets detected by watch connection", func() {
-			written, err := sysLogWriter.Write(standardOutPriority, []byte("just a test"), "App", "2", time.Now().UnixNano())
-			Expect(err).NotTo(HaveOccurred())
-			Expect(written).NotTo(Equal(0))
+		Context("when the drain is slow to consume", func() {
+			BeforeEach(func() {
+				// force an immediate write timeout
+				writeTimeout = -1 * time.Second
```

Comment:
We're guessing that this negative value is used to simulate a timeout without actually having to sleep. If that's the case, could you leave comment explaining the negative value?